openapi.yaml

```diff
@@ -782,11 +782,27 @@ definitions:
   IngredientPatchBody:
     properties:
       data:
```

Comment:
Do you also want to have `data` be required?

Reply:
yep. should be fixed.